.goreleaser.yml

```diff
@@ -290,11 +300,19 @@ brews:
           <key>Label</key>
           <string>com.observiq.collector</string>
 
+          <key>EnvironmentVariables</key>
+          <dict>
+            <key>OIQ_OTEL_COLLECTOR_HOME</key>
+            <string>#{prefix}</string>
+          </dict>
+
           <key>ProgramArguments</key>
           <array>
             <string>#{bin}/observiq-otel-collector</string>
             <string>--config</string>
             <string>#{prefix}/config.yaml</string>
+            <string>--logging</string>
+            <string>#{prefix}/logging.yaml</string>
```

Comment:
This looks good. Should we remove the stdout and stderr redirect that is happening at the bottom of this plist file?